RAG/build_rag.ipynb

```diff
@@ -0,0 +1,88 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n",
+      "Note: you may need to restart the kernel to use updated packages.\n",
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Shell commands for environment and package requirements\n",
+    "%pip install -Uq pip\n",
+    "%pip install -q groq\n",
+    "%pip install -q sqlite-vec"
+   ]
+  },
```

Comment:
It's better practice to install libraries via the terminal and add them to a requirements file. This will make it easier for future teams to install all dependencies from a single file.